docs/userguide/main.md

```diff
@@ -40,10 +40,10 @@ metadata:
 spec:
   general:
     serviceName: my-first-cluster
-    version: 2.3.0
+    version: 2
```

Comment:
Thanks @joshuacox, Since OpenSearch 3.0.0 is the latest version should we change this to 3?